.travis.sh

```diff
@@ -43,6 +43,7 @@ case "$1" in
     # restart the service for the /etc/default/docker change we made after
     # installing the package
     sudo restart docker
+    sleep 5
```

Comment:
I should actually do `docker ps` in a loop up to some time out to actually ensure docker is ready.